src/components/CourtCaseFilter.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, GridCol, GridRow, Link, Select } from "govuk-react"
+import { useState } from "react"
+import { Filter } from "types/CaseListQueryParams"
+
+const queryParamToFilterState = (value: string) =>
+  (value === "triggers" || value === "exceptions" ? value : undefined) as Filter
+
+const CourtCaseFilter = (props: { initialSelection: Filter }) => {
+  const [currentSelection, setCurrentSelection] = useState(props.initialSelection)
+
+  return (
+    <GridRow>
+      <GridCol>
+        <Select
+          label={"Filter cases"}
+          input={{
+            value: currentSelection,
+            onChange: (e) => setCurrentSelection(queryParamToFilterState(e.target.value)),
```

Comment:
I wonder if there is a neat way of using [form submission without JS](https://nextjs.org/docs/guides/building-forms#form-submission-without-javascript) so we don't need to use JS for filtering